pages.de/common/git-send-email.md

```diff
@@ -0,0 +1,24 @@
+# git send-email
+
+> Sende Patches als Email. Möglich für einzelene Commits, Ordner oder .patch Datein.
+> Weitere Informationen: <https://git-scm.com/docs/git-send-email>.
+
+- Sende den letzten Commit der aktuellen Branch:
+
+`git send-email -1`
+
+- Sende einen spezifischen Commmit:
+
+`git send-email -1 {{commit}}`
+
+- Sende die letzten (z.B. 10) Commits der aktuellen Branch:
+
+`git send-email {{-10}}`
+
+- Sende eine Einleitungs Email für eine Reihe von Patches:
+
+`git send-email -{{Anzahl_an_Committs}} --compose`
+
+- Bearbeiten des Email Textes von jedem der Patches:
```

Comment:
Seems to not be an equal translation of the English page.
English version:
`Review and edit the email message for each patch you're about to send:`

Reply:
```suggestion
- Bearbeite den E-Mail Text jedes der zu versendenden Patches:
```